doc/kokkos-promotion.txt

```diff
@@ -74,16 +75,16 @@ Step 2:
        a) serial, openmp, and cuda via the testing scripts in kokkos-kernels/scripts/trilinos-integration (automates the process)
        b) various ATDM-supported builds via Trilinos configuration scripts located in kokkos{-kernels}/scripts/trilinos-integration/ATDM_configurations (not yet automated)
        
-       Run scripts for white (openmp and cuda) and blake (seral) that are provided in kokkos{-kernels}/scripts/trilinos-integration. 
+       Run scripts for automated testing on white (openmp and cuda) and blake (seral) that are provided in kokkos{-kernels}/scripts/trilinos-integration. 
```

Comment:
s/seral/serial/

Reply:
Oop, good catch!

Reply:
I'll clean this up in a future PR since this was merged. A couple more testing configurations need to be added.